### ISSUE TITLE
Regions don't shown anymore on 0.9

### ISSUE THREAD

**Issue by timberman** [overviewer/Minecraft-Overviewer-Addons#16]
Hi there!

The regions.yml is red, but no region is shown on the map. Ist WG2OW broken?

**Comment by pironic** [comment]
you're right that they dont work in .9 anymore. The core overviewer team has not built support for any regions yet. It is on my todo list though...

**Comment by d1rty-pixel** [comment]
@pironic , do you have any news on this? 
The overviewer core does not seem to support regionDatas in the current version (0.12.198-0~overviewer1) - can you please provide a short workaround to get that work again?
